src/App.css

```diff
@@ -1146,22 +1164,51 @@ text {
 /********* Summary Page *********/
 .summary-hero-container {
   display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  justify-content: center;
-  padding: 4em 4em 2em 4em;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  background: linear-gradient(to right, rgba(4, 27, 199, 0.7), rgba(4, 27, 199, 0.8));
+  padding: 4em 4em 4em 4em;
+  clip-path: inset(0% 0%);
+}
+
+.summary-hero-content {
+  z-index: 2;
 }
 
 .summary-hero-container h1 {
   font-size: 60px;
+  color: var(--hero-white);
+  
+}
+
+.summary-hero-container button {
+  background-color: var(--hero-yellow);
+  color: var(--hero-black);
+}
+
+.summary-hero-container button:hover {
+  background-color: var(--hero-yellow-highlighted);
+}
+
+.summary-hero-illustration-container {
+  position: absolute;
+}
+
+.summary-hero-circle {
+  margin-left: calc(100vw - 45em);
+  border-radius: 25% 75% 52% 48% / 69% 44% 56% 31%;
+  background: linear-gradient(to right, var(--hero-yellow), rgba(0,0,0,0.2));
+  min-height: 40em;
+  min-width: 40em;
 }
 
 .summary-page-description {
   font-size: 20px;
 }
 
 .summary-page-content {
-  padding: 0 4em 2em 4em;
+  padding: 0;
```

Comment:
Does this do anything? I'm not sure just asking cause most things default 0 padding I would think